### DISCUSSION TITLE
Multiple grumphp.yml

### DISCUSSION THREAD

**Discussion by a-menshchikov** [phpro/grumphp#964] [Q&A] [answered]
| Q               | A
| --------------- | ---
| Version         | 1.6.0
| Bug?            | no
| New feature?    | no
| Question?       | yes
| Documentation?  |
| Related tickets |

I have merged a few repos into one monorepo and want to keep grumphp.yml files separately, because they have some complicated settings for the same tasks. Is it possible, or the one way to run a few `grumphp` commands with different configs?

**Comment by veewee** [comment, marked as answer]
Yes, you can define pretty much all paths and configurations you want to use in GrumPHP.


You could create your own git hook preset which runs GrumPHP in multiple folders as well.
However, there is no such configuration available out of the box. So you'll need to craft something yourself here.
